HTML/mockup-minimo/style.css

```diff
@@ -0,0 +1,360 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+html {
+    /* 1rem = 10px  10px/16px = 0.625 = 62.5% */
+    font-size: 62.5%;
+}
+
+@media only screen and (max-width: 500px) {
+    html {
+        font-size: 60%;
+    }
+}
+
+body {
+    margin: 0 auto 0 auto;
+    width: 120rem;
```

Comment:
it is possible to remove together with media query that page looks better and sections as footer and sign in was wide page